.gitignore

```diff
@@ -2,13 +2,18 @@
 *.egg-info
 *.eggs
 
+# Editor environment
+.vscode/
+
+
 /dist
 /build
 .tox
 .cache
 .pytest_cache
 .mypy_cache
 *.swp
+*.log
```

Comment:
This can be removed with the `temp.sh` script, because it's not necessary without the script.